### PR TITLE
[TD-2651] Zopim bug fix

### PR DESCRIPTION
**Pull request by raj5036**
### What do you want to achieve?
- Refer #972 

### PR Checklist

- [X] I have tested it locally and all functionalities are working fine.

### How was the code tested?

- Tested locally the main issue has been fixed. 
- Found 2 more bugs while testing, that are independent from this and are on test/prod as well. We will file separate bugs for them.

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/js/app/zendesk-chat-service.js [hunk 1]
```diff
@@ -323,8 +324,13 @@ var onTabClickedHandlerForZendeskConversations = function (event) {
             return member.userId == currentGroupData.metadata.CONVERSATION_ASSIGNEE
         })
         if (!newConversationCreated && assigneeInfo.role != KommunicateConstants.GROUP_ROLE.MODERATOR_OR_BOT) {
+            console.log("currentGroupData ", currentGroupData, assigneeInfo, newConversationCreated);
             newConversationCreated = true;
             Kommunicate.startConversation();
         }
+        // if (document.getElementById('mck-tab-title').innerHTML == MCK_LABELS['conversations.title']) {
+        //     console.log("okoko")
+        //     Kommunicate.startConversation();
```
Comment (gargshantnu) [inline webplugin/js/app/zendesk-chat-service.js:333]: all commented code we can remove?

Comment (raj5036) [reply webplugin/js/app/zendesk-chat-service.js]: done

**Review by gargshantnu — DISMISSED** [review]

**Comment by gargshantnu** [comment]
added @Prochnost55 for review.

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 2]
```diff
@@ -4016,10 +4017,19 @@ var userOverride = {
...
-                                ALStorage.clearMckMessageArray();
+                            if (kommunicate._globals.zendeskChatSdkKey) {
+                                var groupId = result.response[0].id;
+                                $applozic.fn.applozic(
+                                    'loadGroupTab',
+                                    groupId,
+                                    callback
+                                );
+                            } else {
+                                if (MCK_TRIGGER_MSG_NOTIFICATION_TIMEOUT > 0) {
```
Comment (Prochnost55) [inline webplugin/js/app/mck-sidebox-1.0.js:4028]: If msg notification trigger timeout is zero then what should happen?

**Review by gargshantnu — APPROVED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js (hunk 2 above)
Comment (raj5036) [reply webplugin/js/app/mck-sidebox-1.0.js:4028]: Check this PR: https://github.com/Kommunicate-io/Kommunicate-Web-SDK/pull/980/files
This one fixed here
We will merge this branches.

**Pull request closed without merging**